core/src/main/java/hudson/model/ItemGroupMixIn.java

```diff
@@ -329,6 +339,39 @@ public synchronized TopLevelItem createProject( TopLevelItemDescriptor type, Str
     }
 
     /**
+     * Populate a {@link Categories} object from a specific {@link ItemGroup}.
+     *
+     * @return A object that represents a set of {@link Category}
+     */
+    @Restricted(NoExternalUse.class)
+    public static Categories getCategories(Authentication a, ItemGroup c) {
```

Comment:
@jglick this was the [commit](https://github.com/jenkinsci/jenkins/pull/2117/commits/213cf1032beaf8b7744d8c0f4f58564c890eacf3).